src/diamond/handler/influxdbHandler.py

```diff
@@ -29,8 +29,10 @@
 password = root
 database = graphite
 time_precision = s
+influxdb_version = 0.8
```

Comment:
Okay for it to be a string in the conf file, since all values are parsed as strings by default.